source/website/src/views/Step6.vue

```diff
@@ -0,0 +1,659 @@
+                aria-controls="shotTable"
+              ></b-pagination>
+              <br>
+            </div>
+          </b-col>
+        </b-row>
+      </b-container>
+    </div>
+  </div>
+</template>
+
+<script>
+  import Header from '@/components/Header.vue'
+  import Sidebar from '@/components/Sidebar.vue'
+  import {mapState} from "vuex";
+  
+  export default {
+    name: "Step6",
+    components: {
+      Header, Sidebar
+    },
+    data() {
+      return {
+        amc_selector_visible: true,
+        available_amc_instances: [{"endpoint": "","data_upload_account_id": "", "tags": []}],
+        filtered_amc_instances: [],
+        available_amc_instances_fields: [
+          {key: 'actions', label: 'Actions' },
+          {key: 'endpoint', label: 'AMC Endpoint', sortable: true, thStyle: { width: '50%'}},
+          {key: 'tag_list', label: 'Tags', sortable: false},
+        ],
+        totalRows: 1,
+        currentPage: 1,
+        perPage: 5,
+        pageOptions: [5, 10, 15, { value: 100, text: "Show a lot" }],
+        filter: null,
+        filterOn: [],
+        currentPage2: 1,
+        perPage2: 5,
+        currentPage3: 1,
+        perPage3: 5,
+        currentPage1: 1,
+        perPage1: 5,
+        currentPageAmcInstances: 1,
+        perPageAmcInstances: 5,
+        selected_endpoint: '',
+        datasets: [],
+        selected_dataset: '',
+        dataset_fields: [
+          {key: 'selected'},
+          {key: 'dataSetId', sortable:true},
+          {key: 'description'},
+          {key: 'dataSetType', sortable:true},
+          {key: 'fileFormat', sortable:true},
+          {key: 'createdTime', sortable:true},
+          {key: 'updatedTime', sortable:true},
+          {key: 'Actions'}
+        ],
+        etl_jobs: [],
+        etl_fields: [
+          {key: 'DatasetId', label: 'Dataset Id', sortable: true},
+          {key: 'filename', label: 'File Name', sortable: true},
+          {key: 'StartedOn', label: 'Started On', sortable: true},
+          {key: 'ExecutionTime', label: 'Duration', sortable: true},
+          {key: 'JobRunState', label: 'JobRunState', sortable: true},
+          {key: 'show_details', label: 'Show Details', sortable: true}
+        ],
+        uploads: [],
+        upload_fields: [
+          {key: "dateCreated", label: "Date Created", sortable: true},
+          {key: "totalFileCount", label: "Total Files", sortable: true},
+          {key: "errorFileCount", label: "Bad Files", sortable: true},
+          {key: "rowsAcceptedTotal", label: "Rows Accepted", sortable: true},
+          {key: "rowsDroppedTotal", label: "Rows Dropped", sortable: true},
+          {key: "rowsWithResolvedIdentity", label: "Identities Resolved", sortable: true},
+          {key: "sourceFileS3Key", label: "Source File", sortable: true},
+          {key: "status", label: "Status", sortable: true},
+          {key: "show_details", label: "Show Details"}
+        ],
+        isBusy1: false,
```

Comment:
Could these state handling variables be named more uniquely for readability? Like `isBusyGetEtlJobs` or something a little more specific rather than `isBusy2`?